operations.adoc

```diff
@@ -70,18 +71,33 @@ For each final specification:
+** Information on running TCKs in the Jakarta CI infrastructure can be found https://wiki.eclipse.org/TCK:Build_From_Jakarta_EE_TCK_Repo_And_Run#Jenkins_Pipelines[here].
+1. Create a compatibility certification request for the compatible implementation being used to validate the spec in the specification repository issue tracker. If the project does not already have a compatibility-certification-request template, you can use this one: https://github.com/jakartaee/specification-committee/blob/master/compatibility-certification-request.md[compatibility-certification-request.md]
+1. After that TCK is passing, submit a ballot request by creating two https://help.github.com/en/articles/about-pull-requests#draft-pull-requests[draft PRs] requests against the
 https://github.com/jakartaee/specifications[Jakarta EE Specification Committee specifications] repository. The first PR
-provides everything requested in the PR template except the javadoc contents. The second PR would include the javadoc contents.
-These PRs are intended to provide the items that are required to validate the release, and provide the jakarta.ee website content
-for the specification. There is a PR template that guides what to include in the PR.
+provides everything requested in the https://github.com/jakartaee/specification-committee/blob/master/spec_review_checklist.md[PR template] except the javadoc contents. The second PR includes only the apidocs directory with the javadoc contents.
+** These PRs are intended to provide the items that are required to validate the release, and provide the jakarta.ee website content for the specification. The repo has a PR template that lists the expected content for the PR. It includes:
+*** A directory using the specification code as defined in Projects, Specifications, and Documents, e.g., wombat
+*** A subdirectory major.minor corresponding to the version of the spec, (e.g., 1.6), that contains:
+**** Specification Document from (2) above in both pdf and html formats, e.g., wombat_1.6.pdf and wombat_1.6.html
+**** Summary results of TCK run showing at least one compatible implementation
+**** Link to final TCK test bundle if the spec defines a TCK. This will be signed and uploaded to the offical
+specification download area when the ballot passes.
+**** The URL of the OSSRH staging repository for the api, javadoc artifacts
+**** An apidocs directory containing the final JavaDocs from the api build in the second PR.
+1. Update the Jakarta EE API jar by submitting a PR to the https://github.com/eclipse-ee4j/jakartaee-api[jakartaee-api] project that updates the version number of your API jar file.
+1. Update Eclipse GlassFish to use the new version of your API (and implementation, if applicable) by submitting a PR to https://github.com/eclipse-ee4j/glassfish[GlassFish].
+1. For any release other than a service release, create a release record as described in https://www.eclipse.org/projects/handbook/#pmi-commands-release and then:
```

Comment:
On today's Spec call, we determined that we should create these Eclipse Release Records as soon as we realize one is needed. Does this doc talk about the Planning process?  Maybe we could add something there?

For this particular bullet, maybe just change it to "create a release record (if one doesn't already exist) as described.."